src/appBootstrapper.jsx

```diff
@@ -8,21 +8,46 @@ import { computed, extendObservable } from 'mobx';
 import makeRoutes from './routes';
 import lodash from 'lodash';
 import $ from 'jquery';
+import queryString from 'query-string'
 
 // make sure lodash doesn't overwrite (or set) global underscore
 lodash.noConflict();
 
-const routingStore = new ExtendedRoutingStore();
+var routingStore = new ExtendedRoutingStore();
 
-window.routingStore = routingStore;
-
-const stores = {
+var stores = {
     // Key can be whatever you want
     routing: routingStore,
     // ...other stores
 };
 
-const history = syncHistoryWithStore(hashHistory, routingStore);
+var history = syncHistoryWithStore(hashHistory, routingStore);
+
+
+var qs = queryString.parse((window).location.search);
```

Comment:
this code doesn't probably doesn't belong here because it's page specific, but at the same time we want to handle this mapping prior to initial routing.  it's a big complicated.  didn't want to hold it up since it works for purpose of this phase of project.